components/screens/ShoppingCart.js

```diff
@@ -16,15 +66,42 @@ function ShoppingCart({navigation}) {
 }
 
 const styles = StyleSheet.create({
-    container: {
+    container: {        
         flex: 1,
+        paddingTop: 40,
     },
     cart_container: {
-      flex: 1,  
-      backgroundColor: '#fff',
-      alignItems: 'center',
-      justifyContent: 'center',
+        alignItems: 'center',
+    },
+    product_container: {
+        flex: 1,
+        flexDirection: 'row',
+        justifyContent:'space-between',
+    },
+    product_image: {
+        height: 70,
```

Comment:
Instead of defining the height here individually.
Have a base style for the whole app and make the height relative to the base style.
Gets you the uniformity throughout the app.